timelock-agent/src/test/java/com/palantir/atlasdb/timelock/paxos/PaxosRemoteClientsTest.java

```diff
@@ -48,26 +48,24 @@ public class PaxosRemoteClientsTest {
     @Rule
     public final TemporaryFolder temporaryFolder = new TemporaryFolder();
 
-    private PaxosInstallConfiguration paxosInstallConfiguration;
-    private TimeLockInstallConfiguration installConfiguration;
```

Comment:
IntelliJ was flagging these could be locals, so just inlined them while cleaning up this file